ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml

```diff
@@ -2480,6 +2480,7 @@ keys:
           type: str
           valid_values:
           - sampled
+          - hardware
           description: Flow Tracking Type - only 'sampled' supported for now
```

Comment:
The description should be updated. No need to write the possible options. But maybe say that supported values are hardware dependent like we do elsewhere.